elements/box/README.md

```diff
@@ -0,0 +1,37 @@
+The `box` element can contain all the another elements except the `full width` element. The primary function of the `box` element is wrapping its children elements with a HTML table structure because some of the CSS rules only apply on `<table>` or `<td>` elements (mostly on Outlooks).
+
+This wrapper table helps:
```

Comment:
It might also help in grouping elements. I would ephasize that for example these element groups can have the same background if they use boxes.